scripts/hid_configurator/README.rst

```diff
@@ -268,6 +269,23 @@ To obtain information about the firmware running on the device, run the followin
 .. note::
   Only devices with :ref:`nrf_desktop_dfu` support the ``fwinfo`` command.
 
+Getting identification information about the device
+===================================================
+
+To obtain information about the device's Vendor ID, Product ID and generation, run the following command:
+
+.. parsed-literal::
+    :class: highlight
+
+    python3 configurator_cli.py DEVICE devinfo
+
+.. note::
+  Only devices with :ref:`nrf_desktop_dfu` support the ``devinfo`` command.
+
+The command can be used to obtain Vendor ID and Product ID of devices connected via an nRF Desktop dongle.
```

Comment:
This can only be done when connected via dongle?

Reply:
No, but there are other ways to get VID and PID for devices connected directly over USB or BLE (connection through dongle is the reason why we introduced the values to config channel)